package.json

```diff
@@ -32,7 +32,7 @@
     "commander": "2.15.1",
     "console.table": "0.10.0",
     "dotenv": "5.0.1",
-    "eac.js-client": "github:ethereum-alarm-clock/eac.js-client",
+    "eac.js-client": "4.0.2",
```

Comment:
The new package is `@ethereum-alarm-clock/timenode-core` but the versions are the same right now, so this doesn't have to be updated until we release a new version.